server/views/serviceProviderReferrals/referralNavigationTemplate.njk

```diff
@@ -12,6 +13,12 @@
   {{ govukNotificationBanner(serviceUserBannerArgs) }}
   <div class="govuk-grid-row">
     <div class="govuk-grid-column-two-thirds">
+
+        {{ govukBackLink({
```

Comment:
Sorry, approved then realised the indentation isn't quite right (I think) - shoud this not line up with the `<h1>` below? It's really hard to get nunjucks files linted properly... I wonder if there's an easy way to solve that.

Reply:
yeh i couldn't find a plugin that worked well. also i don't really even know the correct formatting for this half the time.